packages/core/src/http-middlewares/after/log-response.js

```diff
@@ -6,15 +6,25 @@ const prepareRequestLog = (req, resp) => {
   req = req || {};
   resp = resp || {};
 
-  let body;
+  let responseBody;
   if (!req.raw) {
     if (typeof resp.content !== 'string') {
-      body = JSON.stringify(resp.content);
+      responseBody = JSON.stringify(resp.content);
     } else {
-      body = resp.content;
+      responseBody = resp.content;
     }
   } else {
-    body = '<probably streaming data>';
+    responseBody = '<probably streaming data>';
+  }
+
+  let requestBody = req.body;
+  if (
+    requestBody &&
+    requestBody.constructor &&
+    requestBody.constructor.name === 'FormData'
```

Comment:
Does this depend on a specific npm package being used? I imagine there's more than one way to generate form data.

Reply:
Good point! Took a more generic approach in 2b7edbc.

Reply:
As a side note, `x instanceof stream` is also how node-fetch tells if the body is a stream: https://github.com/node-fetch/node-fetch/blob/73b7a089742e86132b1b2acc53c6b56169cb57c2/src/body.js#L270